src/main/webapp/js/reducers/auth-reducer.js

```diff
@@ -29,6 +35,18 @@ export default (state = initialState, action) => {
       return { ...state, permissions: action.payload };
     case AUTH_ERROR:
       return { ...state, error: action.payload, authenticated: false };
+    case STORE_LOGIN: {
```

Comment:
I'm not sure if AUTH_USER, SET_COUNTER_LOGOUT_TIME, SET_PERMISSIONS and STORE_LOGIN should be done in separate action types, it's all called in the same place